src/form/Inputs.js

```diff
@@ -41,6 +41,9 @@ export function TicketInput({ name, ...rest }) {
       type="password"
       name={name}
       placeholder={PLACEHOLDER_TICKET}
+      autoCapitalize="none"
```

Comment:
Nice, +1 for accessibility development! But, we haven't paid much attention to that during development at all, and I wouldn't be surprised if there are things in Bridge that are completely messed up for screenreaders and what have you. ):

Reply:
yeah our accessibility is absolute garbage right now — i wasn't paying any attention. you can use chrome's a11y inspector to see that things are broken, especially around form stuff